modules/cnf-topology-aware-lifecycle-manager-backup-recovery.adoc

```diff
@@ -9,6 +9,12 @@
 If an upgrade of a cluster fails, you can manually log in to the cluster and use the backup to return the cluster to its preupgrade state. There are two stages:
 
 Rollback:: If the attempted upgrade included a change to the platform OS deployment, you must roll back to the previous version before running the recovery script.
+
+[IMPORTANT]
+====
+A rollback is only applicable to upgrades from TALM and SNO. This process does not apply to rollbacks from any other upgrade type.
```

Comment:
SNO must be spelled out, as per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#single-node-openshift.

Reply:
good call :) tysm